channels/slack/src/main/kotlin/com/justai/jaicf/channel/slack/SlackChannel.kt

```diff
@@ -38,10 +40,21 @@ class SlackChannel private constructor(
     private constructor(botApi: BotApi, urlPrefix: String): this(botApi) {
         val config = SlackConfig().apply {
             methodsEndpointUrlPrefix = urlPrefix
+            methodsConfig = MethodsConfig().apply {
+                isStatsEnabled = false
```

Comment:
How this option affects requests processing?

Reply:
This disables OAuth authorization on every request, as we don't have OAauth configured in JAICP.